Src/FluentAssertions/Common/MemberPath.cs

```diff
@@ -96,6 +100,19 @@ public bool HasSameParentAs(MemberPath path)
         /// </summary>
         public bool GetContainsSpecificCollectionIndex() => dottedPath.ContainsSpecificCollectionIndex();
 
+        /// <summary>
+        /// Returns a copy of the current object as if it represented an un-indexed item in a collection.
+        /// </summary>
+        public MemberPath WithCollectionAsRoot()
+        {
+            return new MemberPath
```

Comment:
Couldn't we just use the existing ctor `MemberPath(Type reflectedType, Type declaringType, string dottedPath)` instead?

Reply:
Well spotted 👍